package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "resin-semver",
   "version": "0.1.1",
-  "description": "resin specific semver utility methods",
+  "description": "Resin.io specific semver utility methods",
   "main": "src/index.js",
   "typings": "src/index.d.ts",
   "scripts": {
     "test": "mocha -r ts-node/register --outDir tmp test/index.spec.ts",
-    "build": "tsc src/index.ts"
+    "build": "tsc src/index.ts",
+    "ci": "npm run test"
```

Comment:
Normal developers should run that too though, and normal developers running `npm run ci` feels weird. 

Generally, I'd make `npm test` the go-to entry point to test everything - it's nice and intuitive and it's what everybody & everything will run by default - and make `test` run `npm run lint && npm run mocha`.